### DISCUSSION TITLE
Updating other module in monorepo

### DISCUSSION THREAD

**Discussion by mbrevda** [renovatebot/renovate#13918] [Request Help] [answered]
Using npm workspaces, CI is set to only build/publish the sub-repo that was changed (based on file paths). Being that other packages in the monorepo all reference the published module in `package.json` as `file:packages/foo` and not by version number, will the consuming modules be updated? Fwiw, we don't currently commit the version of the updated package to git (we just increment the latest version in npm).

Thanks!

**Comment by rarkins** [comment, marked as answer]
If the reference in package.json is a file-based one then you shouldn't see it updated in any way

**Reply by mbrevda** [reply]
Thanks! Don't suppose you have any ideas how to go about this then? Otherwise how do monorepos work if the packages depend on eacher other?

**Reply by rarkins** [reply]
It should "just work", assuming you're using Lerna or Yarn packages

**Reply by mbrevda** [reply]
I'm using npm, thanks - I'll look in to the others!
